### DISCUSSION TITLE
About a linking issue keyword

### DISCUSSION THREAD

**Discussion by m1sk9** [community/community#43135] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Currently, when using keywords ( `close` or `fix` etc... ) to link pull requests to issues, you need to write each keyword individually for multiple issues.

Example: `close #23, close #34`

I would like this to work with comma separators.

Example: `close #23, #34`

It is very verbose to write keywords one by one

**Comment by LuisContrerasGlz** [comment]
It is currently not possible to use comma separators to link multiple issues to a pull request using keywords. To link multiple issues to a pull request using keywords, you will need to use the keyword for each issue separately.

Hope this helps :-)

**Comment by missionfloyd** [comment]
This is what I came here to suggest.
